### DISCUSSION TITLE
Set default page to list of your repos, sorted by frequency of your commits

### DISCUSSION THREAD

**Discussion by sammcj** [community/community#39244] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It would be really good if you could set your "default" GitHub landing page to show a list of all your repositories - and be able to sort them by how frequently _you_ (not other people / bots) interact with them.

I spend so much time clicking around like a mad man trying to find 'that repo I worked on a lot the other day' but none of the default sort options seem all that useful.

**Comment by jfullstackdev** [comment]
repositories by default are arranged from the last time you updated it, like this 

![image](https://user-images.githubusercontent.com/47092464/202179703-5dce3bd3-2a02-4c6f-b9b1-9244e9032ce4.png)

is that not the one you are asking ?

**Reply by sammcj** [reply]
Mine seems to show repos that were updated by bots etc... and with a lot of padding, so if you work with 20+ repos a day they just get lost in the noise.

![03BAA3D1-9B7B-4741-A11B-3B7F0CCEF84C](https://user-images.githubusercontent.com/862951/202181511-770a4138-9eff-4005-a1a0-356e8f97ce4d.jpeg)

>

**Reply by sammcj** [reply]
There isn't an obvious button to go right to viewing a list of your repos from what I can see either?

**Reply by jfullstackdev** [reply]
it's not, it's your dashboard

this is the link for your repos according to last time you updated it

https://github.com/sammcj?tab=repositories

**Reply by sammcj** [reply]
Perhaps if that "top repositories" on the left had some sort options and could be resized.

**Reply by sammcj** [reply]
Can you set your repositories to the default landing page instead of the dashboard?

Otherwise - I feel like there should be a button to get straight to them from the dashboard.

**Reply by sammcj** [reply]
If I zoom in so it loads the compressed / mobile layout the "repositories" section header isn't a link 🤔 

![BA83B842-B8D6-4854-81B1-3BD0B6FE2B2C](https://user-images.githubusercontent.com/862951/202183136-79876fdd-8faa-4d7b-8107-9ec5ea974961.jpeg)

**Reply by jfullstackdev** [reply]
it's mobile I guess so ? 

I'm using laptop
